tests/unit/executors/encoders/numeric/test_transform_encoder.py

```diff
@@ -1,20 +1,26 @@
 import os
 import pickle
-import shutil
 
+import pytest
 import numpy as np
 
+from jina.executors.metas import get_default_metas
 from jina.executors.encoders.numeric import TransformEncoder
-from tests import rm_files
 
 input_dim = 28
 target_output_dim = 2
 
+@pytest.fixture(scope='function', autouse=True)
+def metas(request, tmpdir):
```

Comment:
As Han suggested, renaming would be nice. What about `test_metas` or `default_test_metas`?

Reply:
sure